Packaging warning

I see this warning when building the packages on Linux

```
~/Documents/icerpc-csharp/tools/IceRpc.Slice.Tools ~/Documents/icerpc-csharp
dotnet pack -nr:false -c Debug
MSBuild version 17.5.0+6f08c67f3 for .NET
  Determining projects to restore...
  All projects are up-to-date for restore.
  IceRpc.Slice.Tools -> /home/jose/Documents/icerpc-csharp/tools/IceRpc.Slice.Tools/src/IceRpc.Slice.Tools/bin/Debug/netstandard2.0/IceRpc.Slice.Tools.dll
/usr/local/share/dotnet-sdk-7.0.202/sdk/7.0.202/Sdks/NuGet.Build.Tasks.Pack/build/NuGet.Build.Tasks.Pack.targets(221,5): warning NU5129: - At least one .props file was found in 'build/', but 'build/IceRpc.Slice.Tools.props' was not. [/home/jose/Documents/icerpc-csharp/tools/IceRpc.Slice.Tools/src/IceRpc.Slice.Tools/IceRpc.Slice.Tools.csproj]
/usr/local/share/dotnet-sdk-7.0.202/sdk/7.0.202/Sdks/NuGet.Build.Tasks.Pack/build/NuGet.Build.Tasks.Pack.targets(221,5): warning NU5129:  [/home/jose/Documents/icerpc-csharp/tools/IceRpc.Slice.Tools/src/IceRpc.Slice.Tools/IceRpc.Slice.Tools.csproj]
  Successfully created package '/home/jose/Documents/icerpc-csharp/tools/IceRpc.Slice.Tools/src/IceRpc.Slice.Tools/bin/Debug/IceRpc.Slice.Tools.0.1.0-preview3.nupkg'.
~/Documents/icerpc-csharp
```

--- a/tools/IceRpc.Slice.Tools/src/IceRpc.Slice.Tools/IceRpc.Slice.Tools.PlatformInfo.props
+++ b/tools/IceRpc.Slice.Tools/src/IceRpc.Slice.Tools/IceRpc.Slice.Tools.PlatformInfo.props
@@ -1,8 +1,0 @@
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-    <PropertyGroup>
-        <OSName Condition="$([MSBuild]::IsOSPlatform('Linux'))">linux</OSName>
-        <OSName Condition="$([MSBuild]::IsOSPlatform('Windows'))">windows</OSName>
-        <OSName Condition="$([MSBuild]::IsOSPlatform('OSX'))">macos</OSName>
-        <OSArch>$([System.Runtime.InteropServices.RuntimeInformation]::OSArchitecture.ToString().ToLower())</OSArch>
-    </PropertyGroup>
-</Project>

